Android/app/src/main/java/com/cs4125/bikerentalapp/view/ConfirmRentFragment.java

```diff
@@ -91,17 +91,18 @@ private void configureUiItems(View view) {
 
 
     private void setBikeStatus(){
-        String username = "roryegan";
-
         LiveData<ResponseBody> liveResponse;
         if(RentOrReturn == 1){
             liveResponse = rentViewModel
-                    .rentBike(username, bikeId, 100);
+                    .rentBike(bikeId, 46);
         }
         else{
-            liveResponse = null;
-        }
-            liveResponse.observe(this, this::observeResponse);
+            liveResponse = null;        }
```

Comment:
It doesn't look like this else really does anything, since liveResponse will already be null, setting it to null won't make any difference. Also if RentOrReturn should have it's first letter lowercase. If there are any other variables or methods named like that make sure to change them.